src/client/src/rt-components/route-wrapper/RouteWrapper.tsx

```diff
@@ -22,14 +24,31 @@ type WindowRole = 'main' | 'sub'
 interface RouteWrapperProps {
   children: ReactNode
   windowType?: WindowRole
+  title?: string
 }
 
-const RouteWrapper: React.FC<RouteWrapperProps> = ({ children, windowType = 'main' }) => {
+const RouteWrapper: React.FC<RouteWrapperProps> = props => {
+  const { children, windowType = 'main', title } = props
   const platform = usePlatform()
+  const { symbol } = useParams()
+
   const { PlatformHeader, PlatformControls, PlatformRoute, window } = platform
+
+  let formattedCurrency
+
+  if (!title && symbol) {
+    formattedCurrency = currencyFormatter(symbol)
+  }
+
```

Comment:
Can't we just pass symbol formatted as title? seems to me that this component shouldn't know about params. What do you think?